conda-envs/environment-dev-py37.yml

```diff
@@ -11,6 +16,7 @@ dependencies:
 - numpy=1.15
 - numpydoc>=0.9
 - pandas=0.24
+- patsy
```

Comment:
I don't think we need patsy anymore as the glm submodule was removed.

Reply:
Removed it